hw2/04-background.js

```diff
@@ -1 +1,35 @@
 // Add your code here
+const button = document.getElementById('button');
+const body = document.getElementById('body');
+let interval = 0;
+
+const startColorChange = (timer) => {
+	console.log('TIME: ', timer);
+	interval = setInterval(changeBackgroundColor, timer * 1000);
+	button.textContent = 'Stop';
+	button.className = 'btn btn-danger mt-3';
+};
+
+function changeBackgroundColor() {
+	let r = Math.floor(Math.random() * 256);
+	let g = Math.floor(Math.random() * 256);
+	let b = Math.floor(Math.random() * 256);
+	let a = Math.random() * 0.5 + 0.5;
+	let color = `rgba(${r},${g},${b},${a})`;
+	body.style.backgroundColor = color;
+}
+
+const stopColorChange = () => {
+	clearInterval(interval);
+	button.textContent = 'Start';
+	button.className = 'btn btn-primary mt-3';
+};
+
+button.addEventListener('click', function () {
```

Comment:
Repeat - AirBnb style guide [(19.1)](https://github.com/airbnb/javascript#whitespace--spaces) recommends using soft tabs (2 spaces) for function indentation.